helm/values.yaml

```diff
@@ -772,6 +773,28 @@ api:
   #      verificationKey: ozhbx5HJCS41NzKrBSQ0vZU1WOmG0Uhm # verificationKey or the JWKS URL
   #      issuer: MY_ISSUER
 
+  integration:
+    enabled: false
+    port: 8072
+    ingress:
+      enabled: false
+      ingressClassName: ""
+      path: /integration-controller(/.*)?
+      pathType: Prefix
+      hosts:
+        - apim.example.com
+      annotations:
+        kubernetes.io/ingress.class: nginx
+        nginx.ingress.kubernetes.io/proxy-read-timeout: 3600                                                                                                                                              │
+        nginx.ingress.kubernetes.io/proxy-send-timeout: 3600
+        nginx.ingress.kubernetes.io/rewrite-target: /$1
+#      tls:
+#        - hosts:
+#            - apim.example.com
+#          secretName: api-custom-cert
```

Comment:
As it is disabled by default, maybe we can keep it as a comment ? This could ensure backward compatibility with existing `values.yml` that not use (yet 😊) the integration section.
WDYT ?

Reply:
Not sure to understand. I'd like to keep the Ingress default config even if it is disabled by default because it is quite specific and it should handle 95% of the usage (especially in our env)